crates/optimism/node/src/node.rs

```diff
@@ -68,7 +69,7 @@ impl OptimismNode {
             .pool(OptimismPoolBuilder::default())
             .payload(OptimismPayloadBuilder::new(
                 compute_pending_block,
-                OptimismEvmConfig::default(),
+                OptimismEvmConfig::new(Arc::new(OpChainSpec { inner: Default::default() })),
```

Comment:
this shouldn't use empty chainspec, we should update `OptimismPayloadBuilder` simialrly to `EthereumPayloadBuilder` to obtain chainspec from `ctx`

Reply:
Done